WooCommerce/Classes/ViewRelated/Dashboard/Settings/About/AboutViewController.swift

```diff
@@ -267,6 +269,51 @@ extension AboutViewController: UITableViewDelegate {
     }
 }
 
+// MARK: - Crash Debug Menu
+//
+extension AboutViewController {
+
+    fileprivate var crashDebugMenuGestureRecognizer: UITapGestureRecognizer {
+        let gestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(didInvokeCrashDebugMenu))
+        gestureRecognizer.numberOfTapsRequired = 4
+        return gestureRecognizer
+    }
+
+    @objc func didInvokeCrashDebugMenu(_ sender: UITapGestureRecognizer? = nil) {
+
+        let menuTitle = NSLocalizedString(
+            "Crash Debug Menu",
+            comment: "The title for a menu that helps debug crashes in production builds"
+        )
+
+        let crashDebugMenu = UIAlertController(title: menuTitle, message: nil, preferredStyle: .actionSheet)
+        crashDebugMenu.addAction(crashDebugMenuCrashAction)
+        crashDebugMenu.addAction(crashDebugMenuCancelAction)
+
+        present(crashDebugMenu, animated: true, completion: nil)
+    }
+
+    private var crashDebugMenuCrashAction: UIAlertAction {
+        let crashTitle = NSLocalizedString(
+            "Crash Immediately",
+            comment: "The title for a button that causes the app to deliberately crash for debugging purposes"
+        )
+
+        return UIAlertAction(title: crashTitle, style: .destructive) { _ in
+            CrashLogging.crash()
+        }
+    }
+
+    private var crashDebugMenuCancelAction: UIAlertAction {
+        let cancelTitle = NSLocalizedString(
+            "Cancel",
+            comment: "The title for a button that dismisses the crash debug menu"
+        )
+        return UIAlertAction(title: cancelTitle, style: .cancel) { _ in
+            // Do nothing – the alert will still be dismissed
+        }
```

Comment:
We can also pass a `nil` handler. 🙂 

```suggestion
        return UIAlertAction(title: cancelTitle, style: .cancel, handler: nil)
```